### PR TITLE
Clean up /var/tmp/ when using oci-archives when creating containers 

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
Temporary files created when dealing with images in /var/tmp will automatically be cleaned up on reboot.
```
[NO NEW TESTS NEEDED]

### CODE REVIEW

**Comment by rhatdan** [comment]
Replaces: https://github.com/containers/podman/pull/19201

File: contrib/tmpfile/podman.conf
Comment (Luap99) [inline contrib/tmpfile/podman.conf:11]: Why only at boot? I feel like if you have a long running server we can clean these up much earlier. I would even say set this to 1 day.

Comment (rhatdan) [reply contrib/tmpfile/podman.conf:11]: Well then you would have a risk of someone pulling an image and then systemd kills it.
These should only leak when something is killed, so it should not be that common.

Comment (Luap99) [reply contrib/tmpfile/podman.conf:11]: No that is not what tmpfiles does. It will only removes files that are not touched in the last X time.
> The date field, when set, is used to decide what files to delete when cleaning. If a file or directory is older than the current time minus the age field, it is deleted.

So I don't see how systemd would ever delete files that are still in use by us.

Comment (rhatdan) [reply contrib/tmpfile/podman.conf:11]: I don't see how you can use a timer with a glob to remove files?

**Comment by rhatdan** [comment]
Use this PR rather then https://github.com/containers/podman/pull/19305

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL
@edsantiago @Luap99 @giuseppe @vrothberg @TomSweeneyRedHat @mheon @flouthoc PTAL

File: contrib/tmpfile/podman.conf
Comment (Luap99) [reply contrib/tmpfile/podman.conf:11]: Yeah not 100% sure either, I don't have time to look into it but this change works as well so fine for me.

**Review by Luap99 — COMMENTED** [review]
LGTM

**Review by flouthoc — APPROVED** [review]
LGTM
/lgtm
/approve
/hold

**Comment by rhatdan** [comment]
For some reason, containers/image would not update until I vendored the universe.

**Pull request merged**
